src/js/api/config.js

```diff
@@ -61,9 +61,50 @@ function _adjustDefaultBwEstimate(estimate) {
     return Defaults.bandwidthEstimate;
 }
 
+function _copyToLocalization(allOptions) {
+    const { advertising, related, sharing } = allOptions;
+    const localization = Object.assign({}, Defaults.localization, allOptions.localization);
```

Comment:
We shouldn't apply the localizations to the `Defaults.localization`, it would break the translation process.